docs/firstobject.md

```diff
@@ -16,40 +17,39 @@ Note: This guide will assume Archipelago is running on `http://localhost:8001`,
 
 Start by opening `http://locahost:8001` in your favourite Web Browser.
 
-![Welcome](../imgs/Welcome.jpg)
+![Welcome](../imgs/ingest-first-object_beta2/01_log-in.jpg)
```

Comment:
@mitchellkeaney sorry man. I know you are learning this but this is the only way to learn. Having a folder that is using the name of the branch is totally discouraged and not needed. The branch itself is already taking care of that by keeping track of its own files. What i'm saying is all files just need to go into ../imgs/ and no ingest-first-object_beta2/. You could add something that says ingest-first-object if you want to keep images separated for each doc, and in that case it would even be better if you used the same name as the file, so imgs/firstobject/.  So please rename that folder or move things out of there back to imgs. thanks

Reply:
@DiegoPino makes sense! thanks for the tip

Reply:
great!